setup.py

```diff
@@ -11,14 +11,14 @@
 setuptools.setup(
     name="mewbot",
     version="0.0.1",
-    author="Benedict Harcout $ Alex Cameron",
-    author_email="[Ben public email for this project] & mewbot@quicksilver.london",
+    author="Benedict Harcourt & Alex Cameron",
+    author_email="mewbot@tea-cats.co.uk & mewbot@quicksilver.london",
     description="Light weight text based generic irc framework",
```

Comment:
Should be "YAML-driven Bot Framework", or something similar?

Reply:
I definitely agree. Will update.